doc/architecture/decisions/0023-BLAST-species-sequence-interactions.md

```diff
@@ -0,0 +1,33 @@
+# 23. BLAST Species/Sequence interactions
+
+Date: 2022-10-17
+
+## Status
+
+Draft
+
+## Context
+This document summarises the interactions between the input sequences and selection of species.
+
+## Decision
+
+To ensure fair usage we have applied a limit of 30 sequences to be submitted in a single submission. 
```

Comment:
- 30 is a wrong number as of now
- This doesn't ensure fair usage. There's nothing stopping the user from submitting another batch of jobs right after the first one

Reply:
1. I will update the limit to 50.
2. I am not sure how to rephrase it. We cannot say we are making it hard for users :-) 
3. Will update it as 
```
export const MAX_BLAST_SPECIES_COUNT = 25;
export const MAX_BLAST_SEQUENCE_COUNT = 50;
```